drizzlepac/haputils/poller_utils.py

```diff
@@ -768,9 +771,30 @@ def build_poller_table(input, log_level, poller_type='svm'):
             # This code interprets that as False since it is NOT new, so the code
             # inverts the meaning from the pipeline poller file.
             if poller_type == 'mvm':
+                # Translate new format back to old format ("NEW" -> 0 and "OLD" -> 1)
+                poller_table_mapping = {"NEW": 0, "OLD": 1}
+                for tbl_ctr in range(0, len(input_table)):
+                    if input_table[tbl_ctr]['skycell_new'] in ["OLD", "NEW"]:
```

Comment:
There should be logic to trap the case where a 'corrupted' (hand-generated?) file was provided with mixed-case values or perhaps different strings other than 'OLD' and 'NEW' (all upper-case)?  Mixed case inputs can be handled by simply using '.upper()'.  Other problems, though, should immediately raise an Exception so that the user can edit the file to conform to supported values which get reported in the Exception ('We only support values of "OLD" and "NEW"...).

Reply:
Issue resolved in commit [66de19a](https://github.com/spacetelescope/drizzlepac/pull/874/commits/66de19adca258a7c5da14aec0846878d3d8c147e)